setup/artifacts/http.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: single
+title:  "Configuring HTTP Artifact Credentials"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that read data from artifacts can read HTTP files directly.
+
+If the files are hidden behind basic auth, you can configure an artifact
+account with the needed credentials to read your artifact. _If not_, no further
+configuration is needed, Spinnaker will automatically add a
+`no-auth-http-account` for this purpose.
+
+## Prerequisites
+
+Collect your basic auth `$USERNAME` and `$PASSWORD`, pick a
+`$USERNAME_PASSWORD_FILE` location on your disk, and run:
+
+```bash
+echo ${USERNAME}:${PASSWORD} > $USERNAME_PASSWORD_FILE
+```
+
+## Editing Your Artifact Settings
+
+Collect the `$USERNAME_PASSWORD_FILE` value from the
+[prerequisites](#prerequisites) section above.
+
+First, make sure that artifact support is enabled:
```

Comment:
Same with these: three steps, and step two shouldn't start with "First, ..." ;)

1. Collect the...
2. Make sure the artifact...
(command)
3. Add an artifact account
(command)